modules/gdnative/nativescript/nativescript.cpp

```diff
@@ -315,15 +315,15 @@ void NativeScript::get_script_signal_list(List<MethodInfo> *r_signals) const {
 bool NativeScript::get_property_default_value(const StringName &p_property, Variant &r_value) const {
 	NativeScriptDesc *script_data = get_script_desc();
 
-	Map<StringName, NativeScriptDesc::Property>::Element *P = NULL;
+	OrderedHashMap<StringName, NativeScriptDesc::Property>::Element P;
 	while (!P && script_data) {
```

Comment:
Why/how does this work? 😄 P now is stack allocated and a value type, not a pointer anymore, so I don't really know what the `!P` is still doing here